std-bits/base/src/main/java/org/enso/base/Text_Utils.java

```diff
@@ -186,6 +186,20 @@ public static boolean contains(String string, String substring) {
     return searcher.first() != StringSearch.DONE;
   }
 
+  /**
+   * Combine strings {@code strings}
+   *
+   * @param strings the string values to be concatenated
+   * @return the resulting string
+   */
+  public static String concat(String... strings) {
+    StringBuilder sb = new StringBuilder();
+    for (String arg: strings) {
+      sb.append(arg);
+    }
+    return sb.toString();
+  }
```

Comment:
I'm not quite sure if this is necessary. Our `+` operator uses ropes under the hood to achieve very good amortized complexity of string concatenation.